orttraining/tools/ci_test/run_bert_perf_test.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import subprocess
+import sys
+import os
+from collections import namedtuple
+
+SCRIPT_DIR = os.path.realpath(os.path.dirname(__file__))
+
+def parse_args():
+  parser = argparse.ArgumentParser(description="Runs BERT performance tests.")
+  parser.add_argument("--binary_dir", required=True,
+                      help="Path to the ORT binary directory.")
+  parser.add_argument("--training_data_root", required=True,
+                      help="Path to the training data root directory.")
+  parser.add_argument("--model_root", required=True,
+                      help="Path to the model root directory.")
+  return parser.parse_args()
+
+# using the same params from "GitHub Master Merge Schedule" in OneNotes
+def main():
+    args = parse_args()
+
+    Config = namedtuple('Config', ['name', 'use_mixed_precision', 'max_seq_length', 'batch_size', 'max_predictions_per_seq'])
+    configs = [
+        Config('fp16-128', True, 128, 66, 20),
```

Comment:
nit: as it is, name could be generated from use_mixed_precision and max_seq_length